README.md

```diff
@@ -103,6 +103,40 @@ Support for Go versions on the OpenTelemetry Collector is updated as follows:
 
 Official OpenTelemetry Collector distro binaries may be built with any supported Go version.
 
+## Verifying the images signatures
+
+> **Note**: To verify a signed artifact or blob, first [install Cosign](https://docs.sigstore.dev/system_config/installation/), then follow the instructions below.
+
+We are signing the images `otel/opentelemetry-collector` and `otel/opentelemetry-collector-contrib` using [sigstore cosign](https://github.com/sigstore/cosign) tool and to verify the signatures you can run the following command:
+
+```console
+$ cosign verify \
```

Comment:
Here's the command I executed:

```
cosign verify \
  --certificate-identity=https://github.com/open-telemetry/opentelemetry-collector-releases/.github/workflows/base-release.yaml@refs/tags/v0.95.0 \
  --certificate-oidc-issuer=https://token.actions.githubusercontent.com \
  ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector:0.95.0
```

Note the extra `\` at the end of line 114.

Reply:
@cpanato, I think this is the only thing missing to merge this PR :-)